documentation/book/proc-adding-users-the-strimzi-admin-role.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// assembly-getting-started-strimzi-admin.adoc
+
+[id='proc-adding-users-the-strimzi-admin-role-{context}']
+= Deploying the User Operator using the Cluster Operator
+
+.Prerequisites
+
+* {ProductName} `CustomResourceDefinitions` installed
+
+.Procedure
+
+. Create the `strimzi-admin` cluster role in {ProductPlatformName}.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f install/strimzi-admin
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f install/strimzi-admin
+
+. Assign the `strimzi-admin` cluster role to an existing {ProductPlatformName} user(s).
```

Comment:
_Please reword as follows:_

Assign the `strimzi-admin` role to one or more existing users in the {ProductPlatformName} cluster.